docs/model-types.mdx

```diff
@@ -0,0 +1,467 @@
+|companyId|jobType       |degree     |major    |industry|yearsExperience|milesFromMetropolis|salary|
++---------+--------------+-----------+---------+--------+---------------+-------------------+------+
+|COMP37   |CFO           |MASTERS    |MATH     |HEALTH  |10             |83                 |130   |
+|COMP19   |CEO           |HIGH_SCHOOL|NONE     |WEB     |3              |73                 |101   |
+|COMP52   |VICE_PRESIDENT|DOCTORAL   |PHYSICS  |HEALTH  |10             |38                 |137   |
+|COMP38   |MANAGER       |DOCTORAL   |CHEMISTRY|AUTO    |8              |17                 |142   |
+|COMP7    |VICE_PRESIDENT|BACHELORS  |PHYSICS  |FINANCE |8              |16                 |163   |
++---------+--------------+-----------+---------+--------+---------------+-------------------+------+
+```
+
+Here, the features are `companyId`, `jobType`, `degree`, `major`, `industry`, `yearsExperience`, and `milesFromMetropolis`.
+
+And the target variable is `salary`.
+
+Let's create and train an ML model using this training dataset.
+
+```sql
+CREATE MODEL salary_predictor
+FROM files
+    (SELECT * FROM salary_dataset)
+PREDICT salary;
+```
+
+On execution, we get:
+
+```sql
+Query successfully completed
+```
+
+### Progress
+
+Here is how to check whether the training process is completed:
+
+```sql
+SELECT *
+FROM mindsdb.models
+WHERE name = 'salary_predictor';
+```
+
+Once the status is `complete`, the training phase is completed.
+
+### Validation Phase
+
+The `CREATE MODEL` statement performs validation of the model.
+
+Additionally, we can validate the model manually by querying it and providing the feature values in the `WHERE` clause like this:
+
+```sql
+SELECT salary, salary_explain
+FROM mindsdb.salary_predictor
+WHERE companyId = 'COMP37'
+AND jobType = 'MANAGER'
+AND degree = 'DOCTORAL'
+AND major = 'MATH'
+AND industry = 'FINANCE'
+AND yearsExperience = 5
+AND milesFromMetropolis = 50;
+```
+
+On execution, we get:
+
+```sql
++------+------------------------------------------------------------------------------------------------------------------------------------------+
+|salary|salary_explain                                                                                                                            |
++------+------------------------------------------------------------------------------------------------------------------------------------------+
+|128   |{"predicted_value": 128, "confidence": 0.67, "anomaly": null, "truth": null, "confidence_lower_bound": 109, "confidence_upper_bound": 147}|
++------+------------------------------------------------------------------------------------------------------------------------------------------+
+```
+
+By comparing the real salary values for the defined individuals and the predicted salary values, one can figure out the accuracy of the ML model.
+
+<Note>
+Please note that MindsDB calculates the model's accuracy while running the `CREATE MODEL` statement.
```

Comment:
> Please note that by default [...]

Because it's not guaranteed that all ML engines will do this.

Reply:
Changed:

> Please note that MindsDB calculates the model's accuracy by default while running the `CREATE MODEL` statement. However, it is not guaranteed that all ML engines do this.